lib/invoker/power/url_rewriter.rb

```diff
@@ -27,6 +28,12 @@ def extract_host_from_domain(complete_path)
 
       private
 
+      def dev_match_regex
+        # [/([\w.-]+)\.dev(\:\d+)?$/, /([\w-]+)\.dev(\:\d+)?$/]
+        tld = Invoker.tld
+        [/([\w.-]+)\.#{tld}(\:\d+)?$/, /([\w-]+)\.#{tld}(\:\d+)?$/]
```

Comment:
@kgrz I'll be adding a validation for tld. Probably something like
```ruby
class Cli < Thor
  def setup
    validate_tld(options[:tld]) if options[:tld]
    # ...
  end

  private

  def validate_tld(tld)
    unless /^[a-z]+$/ =~ tld
      raise InvalidTldError.new('TLD can only contain lower case alphabets')
    end
  end
end
```
This pull is currently a work in progress. I'll tag you or Hemanth for a review once I'm done.